docs/userguide/mng-storage-driver.md

```diff
@@ -0,0 +1,145 @@
+<!--[metadata]>
++++
+title = "Choose a storage driver"
+description = "Learn how select the proper storage driver for your container."
+keywords = ["container, storage, driver, AUFS, btfs, devicemapper,zvfs"]
+[menu.main]
+parent = "mn_use_docker"
++++
+<![end-metadata]-->
+
+# Choose a storage driver
+
+When you start the Docker `daemon`, you can specify which storage driver you want to use with your containers. A container's storage driver can affect the performance a containerized applications. So it is important to understand the different storage options available and select the right one for your application.
```

Comment:
I suggest we change this to the following - "When you start the Docker daemon, you can specify which storage driver to use. Your choice of storage driver can affect the performance of your containerized applications. So it's important to understand the different storage driver options and select the right one for your requirements". 

I personally wouldn't put the word "daemon" in code font - may be hyperlink it to some documentation on the daemon, but not code font.